### DISCUSSION TITLE
Login - can't have more than one account in visual studio

### DISCUSSION THREAD

**Discussion by dsolteszopyn** [community/community#56125] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

In github copilot for visual studio you can manually login and link to a specific account which may or may not be the same account you are signed into visual studio for source control.  With copilot chat it only can uses one github account that is signed into Visual studio.  My paid copilot account isn't the same account i use to sign into visual studio for source control.  Need able to sign into copilot chat like you can with just copilot and not have it directly use the account signed into visual studio.

**Comment by Echostorm44** [comment]
Same problem!  This is really bumming me out, why not do the chat extension the same way?
